python/lsst/meas/extensions/trailedSources/NaivePlugin.py

```diff
@@ -166,8 +167,8 @@ def measure(self, measRecord, exposure):
         ra, dec = self.computeRaDec(exposure, xc, yc)
 
         if measRecord.getShapeFlag():
-            self.log.warning("Shape flag is set for measRecord: %s. Trail measurement "
-                             "will not be made.", measRecord.getId())
+            self.log.debug("Shape flag is set for measRecord: %s. Trail measurement "
+                           "will not be made.", measRecord.getId())
```

Comment:
Can this log message be a bit more specific - which values will be set to nan, e.g.? Just length? Others also?

Reply:
Added explanation that all trail values will be nan if the flag is set.